cmd/syncthing/gui.go

```diff
@@ -210,15 +212,63 @@ func (s *apiSvc) Serve() {
 		ReadTimeout: 10 * time.Second,
 	}
 
+	s.fss = newFolderSummarySvc(s.model)
+	defer s.fss.Stop()
```

Comment:
This might change behaviour, as fss was only stopped on Stop, now it's stopped on listener error.